packages/api/src/storage.ts

```diff
@@ -120,4 +134,39 @@ export class LocalApiStore {
     fs.writeFileSync(this._messagesPath, JSON.stringify(messages))
     return tmpMessage
   }
+
+  async getAudio(messageId: string) {
+    const message = await this.getMessage(messageId)
+    if (!message) {
+      return // TODO: error handle
+    }
+    const filePath = message.file_path
+    if (!filePath) {
+      return // TODO: error handle
+    }
+
+    const audio = fs.readFileSync(path.join(this._audioDir, filePath), 'utf8')
+    if (!audio) {
+      return // TODO: error handle
+    }
+    return audio
+  }
+
+  async putAudio(messageId: string, audio: Api.AudioFile) {
+    const messages = JSON.parse(fs.readFileSync(this._messagesPath, 'utf8'))
+
+    // Get our message to the messages array.
+    const tmpMessage = messages.messages[messageId]
+    if (!tmpMessage) {
+      return // TODO: error handle
+    }
+    // Set file_path
+    tmpMessage.file_path = tmpMessage.uuid + '.ogg'
```

Comment:
Don't know how we want to handle this internally. I just hardcoded in `ogg`'s as per @connorjclark.

Reply:
works for me